### ISSUE TITLE
NVDA does not read nested description/definition lists as such.

### ISSUE THREAD

**Issue by haleyb128** [nvaccess/nvda#10361]
### Steps to reproduce:
Have NVDA read out any nested `<dl>s`.  
JS Bin: https://jsbin.com/musivovati/edit?html,output
### Actual behavior:
When `<dl>`s are read, they are read as lists, which is the first problem, but this has been brought up in a previous issue. While this is still true, it is also a problem that nested `<dl>`s are not read out as such, so the relationships between key-value pairs are not expressed.
### Expected behavior:
`<dl>`s should be read out as such and nested `<dl>`s should express parent/child relationships as well as reading out as `<dl>`s.
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2019.1.1
#### Windows version:
10 Pro Build 18362
#### Name and version of other software in use when reproducing the issue:
Chrome 77.0.3865.90 and Firefox 69.0.1 (64 bit for both)

#### Does the issue still occur after restarting your PC?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes. The same problems have ocurred.

**Comment by haleyb128** [comment]
There is also issue #3858 but that seems like a somewhat different, less important issue.

**Comment by leonardder** [comment]
I consider #3858 to be an issue that talks about issues with definition lists in general. Therefore I'd say this issue is a duplicate of #3858. WOuld you mind if I close this issue and we continue the discussion in #3858?
